Resources/Prototypes/_L5/Catalog/Fills/Crates/food.yml

```diff
@@ -0,0 +1,107 @@
+- type: entity
+  id: CrateFoodProduce
+  parent: CrateFreezer
+  name: produce crate
+  description: a freezer full of random, fresh-picked produce
+  components:
+  - type: StorageFill
```

Comment:
Would you mind redoing this with entity tables? See e.g., https://github.com/lagrange14/substations/blob/main/Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml#L6.